source/systems/gaea_user_guide.rst

```diff
@@ -1290,6 +1345,210 @@ For SHMEM codes, users must load the xt-shmem module before compiling:
 
   $ module load xt-shmem
 
+Debugging with DDT
+==================
+
+Arm DDT is a powerful, easy-to-use graphical debugger. With Arm DDT it
+is possible to debug:
+
+* Single process and multithreaded software
```

Comment:
The double `*` is not the correct format for a nested <ul> in RST.

Reply:
fixed thanks.